skytemple_files/graphics/chara_wan/dbg/test.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import os
+import shutil
+import math
+import glob
+import chara_wan.writer as exWriter
+
+
+
+def getWanFilePath(baseDir, dirName, index):
+    inDir = os.path.join(baseDir, dirName)
+    inFile = None
+    for file in glob.glob(os.path.join(inDir, '*_' + format(index, '04d') + '_*')):
+        inFile = file
+    return inFile
+
+def ConvertJoinSeparateChara(baseDir, index, anim_name_map, sdwImg):
+    monsterPath = getWanFilePath(baseDir, 'monster', index)
+    m_groundPath = getWanFilePath(baseDir, 'm_ground', index)
+    m_attackPath = getWanFilePath(baseDir, 'm_attack', index)
+
+    print(format(index, '04d'))
```

Comment:
Please remove prints and replace them with a logger instead.
To add a logger, just add the following to the very top of each module you need one:
```py
import logging
logger = logging.getLogger(__name__)
```

Reply:
Nevermind, print is okay in the dbg script.